homeassistant/components/wiz/__init__.py

```diff
@@ -60,6 +60,15 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         await bulb.async_close()
         raise ConfigEntryNotReady(f"{ip_address}: {err}") from err
 
+    if bulb.mac != entry.unique_id:
```

Comment:
Interesting case, I guess this could happen in more integrations at times

Reply:
I have the same guarding in `flux_led`.  I believe Otto recently added similar guarding to esphome